programs/manifest/src/program/instruction_builders/batch_update_instruction.rs

```diff
@@ -1,12 +1,19 @@
+#[cfg(feature = "certora")]
```

Comment:
Is there any alternative to having these all over the place now?

Reply:
I got rid of a bunch of them.
Might be able to get rid of a few more if we are alright expanding scope of some functions, but dont really want to do that